<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QDialog" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>810</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>810</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>810</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deposit Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #434343;
}</string>
  </property>
  <widget class="QLabel" name="label_amount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_term">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Срок размещения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_percent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_amount">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_term">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_percent">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>месяцев</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tax">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Налоговая ставка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_tax">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_pay">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>205</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    background-color: #DCDCDC;
    color:  #434343;
    border: 1px solid #434343;
    padding: 5px;
	margin: 5px;
    font-size: 13px;
}

QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 1px solid  #434343;
}

QRadioButton::indicator:checked {
    background-color: #F6872E;
    border: 1px solid #434343;
}

QRadioButton::indicator:unchecked {
    background-color: #ffffff;
}
</string>
   </property>
   <property name="text">
    <string>Выплачивать проценты</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_cap">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>255</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    background-color: #DCDCDC;
    color:  #434343;
    border: 1px solid #434343;
    padding: 5px;
	margin: 5px;
    font-size: 13px;
}

QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border-radius: 10px;
    border: 1px solid  #434343;
}

QRadioButton::indicator:checked {
    background-color: #F6872E;
    border: 1px solid #434343;
}

QRadioButton::indicator:unchecked {
    background-color: #ffffff;
}
</string>
   </property>
   <property name="text">
    <string>Добавлять ко вкладу</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pay_or_cap">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 13px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Периодичность</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_frequency">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    color: #434343;
    font-size: 16px;
    font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
    selection-background-color: #F6872E;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #434343;
    border-left-style: solid;
}
</string>
   </property>
   <item>
    <property name="text">
     <string>Ежемесячно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежеквартально</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежегодно</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_add">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Пополнения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_withdrawals">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Снятия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_add_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_add_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_add_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_withdrawals_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_withdrawals_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_withdrawals_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>560</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_equal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color:#ffffff;
	font-size: 16px;
	font-weight: bold;
    background-color: #F6872E;
    border: 1px solid #434343;
    padding: 10px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #D17127, stop: 1 #F6872E);
}</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_interest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tax_amount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Сумма налога</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_output_amount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>760</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #CECECE;
    border: 1px solid #434343;
    padding: 10px;
}</string>
   </property>
   <property name="text">
    <string>Сумма на вкладе</string>
   </property>
  </widget>
  <widget class="QLabel" name="line_interest">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>660</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="line_tax_amount">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>710</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="line_output_amount">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>760</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color:#434343;
	font-size: 16px;
	font-weight: 440;
    background-color: #DCDCDC;
    border: 1px solid #434343;
	text-align: left;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>pictures/kot2.jpg</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
